docs/ChangeLog.md

```diff
@@ -21,10 +20,17 @@ This is the 2.4.0 release. The Splunk Operator for Kubernetes is a supported pla
 
 * CSPL-2459: Upgraded operator-sdk to 1.31.0
 
+* CSPL-2409: Splunk Operator for Kubernetes reorders environment variables
+
 * Git-1223: Fixed an issue in Storage Class for splunk operator helm chart
 
 * Git-1151: fixed indexer deletion in upgrade scenario
 
+* Fixed auth proxy service selector labels
+
+* host multiple splunk operator helm charts on github.io
```

Comment:
Nit: Is this associated with a CSPL? Can we start the sentence with a capital letter? `Host multiple splunk operator helm charts on github.io`